src/categorical_algebra/JSONCSetTransformations.jl

```diff
@@ -0,0 +1,114 @@
+""" JSON serialization of acset transformations.
+"""
+module JSONCSetTransformations
+export generate_json_fin_function, parse_json_fin_function,
+  read_json_fin_function, write_json_fin_function,
+  generate_json_acset_transformation, parse_json_acset_transformation,
+  read_json_acset_transformation, write_json_acset_transformation
+
+import JSON
+using DataStructures: OrderedDict
+
+using ..FinSets, ..CSets
+
+# ACSetTransformation serialization
+#####################
+
+""" Generate JSON-able object representing a FinFunction.
+
+Inverse to [`parse_json_fin_function`](@ref).
+"""
+function generate_json_fin_function(F::FinFunction)
+  OrderedDict{Symbol,Any}(
+    :dom        => dom(    F),
+    :codom      => codom(  F),
```

Comment:
What is going on with this white space?

Reply:
Refactoring rendered this formatting unnecessary for readability.

Reply:
This has been amended in a more recent commit.